build-tools/installers/create-installers.targets

```diff
@@ -145,8 +145,8 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.Cecil.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.Diagnostics.dll" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.Diagnostics.pdb" />
-    <_MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.Generator.dll"  ExcludeFromAndroidNETSdk="true" />
-    <_MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.Generator.pdb"  ExcludeFromAndroidNETSdk="true" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.Generator.dll" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.Generator.pdb" />
```

Comment:
@jpobst was it intentional to leave this file out for .NET 6? in: 91970b17fc948197f97331adbac9af0a46d7bf9b

Reply:
I figured it should match `generator` itself, since that is what uses it:

https://github.com/xamarin/xamarin-android/blob/master/build-tools/installers/create-installers.targets#L125-L126

But yes, anything that provides `generator` needs to provide `Java.Interop.Tools.Generator.dll` as well.

Reply:
Ah, ok we needed `generator.dll` so that is why `generator.exe` was excluded:

https://github.com/xamarin/xamarin-android/blob/f029ab709711d7b4891f5c31a3803a0cb6adcd48/build-tools/create-packs/Microsoft.Android.Sdk.proj#L77-L78

So this should be fine. 👍 

I am very confused to why some binding projects work, though...

Reply:
Maybe it lazy loads the assembly, which would only be used if you were using our enumification stuff.

Reply:
`dotnet new android-bindinglib` creates `EnumsFields.xml` and `EnumMethods.xml`, so that must be what breaks:

https://github.com/xamarin/xamarin-android/blob/f029ab709711d7b4891f5c31a3803a0cb6adcd48/src/Microsoft.Android.Templates/android-bindinglib/Transforms/EnumFields.xml#L1-L14